caddyfile/parse_test.go

```diff
@@ -214,6 +214,36 @@ func TestParseOneAndImport(t *testing.T) {
 			"dir3": 1,
 		}},
 
+		{`localhost:1234{
+		    dir1 foo bar
+		    dir2
+		  }`, true, []string{
+			"localhost:1234{",
+		}, map[string]int{
+			"dir1": 3,
+			"dir2": 1,
+		}},
+
+		{`"localhost:1234"{
```

Comment:
Do you think this should be an error? 🤔 It looks untidy...

Reply:
You mean because of the missing space?
I assume here is intended with the double quote. but we can make this case invalid too.

Reply:
> You mean because of the missing space?

Correct.

I think technically this is one token... but, I guess, now that we talk about it, I have never really thought about this before. If you have an input like `"a" b` then that is clearly 2 tokens, where the first token is interpreted literally. If you have an input like `"a"b`... should that be one or two?

I'm inclined to say that the lexer _should_ lex it as 1 token, where the quoted part is literal and the b is still part of the same token (similar to bash shell, I believe).

So, I think it should be treated the same as the case for `localhost:1234{`.

If that is kind of complex or gnarly, though, maybe we need to patch the lexer on that matter instead.